metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_lineage_v2.py

```diff
@@ -42,25 +60,6 @@ class SnowflakeColumnId:
     objectDomain: Optional[str] = None
 
 
-@dataclass(frozen=True)
-class SnowflakeColumnFineGrainedLineage:
-    """
-    Fie grained upstream of column,
-    which represents a transformation applied on input columns"""
-
-    inputColumns: FrozenSet[SnowflakeColumnId]
-    # Transform function, query etc can be added here
-
-
-@dataclass
-class SnowflakeColumnUpstreams:
```

Comment:
were these just unused?

Reply:
`SnowflakeColumnFineGrainedLineage` was used and created for future extension. This is placeholder to capture "[transformOperation](https://github.com/datahub-project/datahub/blob/master/metadata-models/src/main/pegasus/com/linkedin/dataset/FineGrainedLineage.pdl#L36)" - >entire query or column to column transformation logic along with upstream columns.

Curious - do we get column to column transformation logic in sql parsing ?

Reply:
Ah, I just saw that it was unnecessary so I removed it... did miss the comment in the class sorry. I want to get this out, so can we just add this back when we're passing the query / transformation logic